backend/core/unified_retriever.py

```
@@ -18,143 +18,33 @@
 
-# Prefer the newer Chroma package
-try:
-    from langchain_chroma import Chroma
-except ImportError:
-    # Fallback to community version if new package not available
-    from langchain_community.vectorstores import Chroma
-
-from ..ingest.chunking import splitter_for_ext
-from ..ingest.loaders import load_doc
-from .llm_utils import extract_topics_with_llm
+from .config import AppConfig
```

Ruff (0.12.2)

`.config.AppConfig` imported but unused

Remove unused import: `.config.AppConfig`

(F401)